crates/turbopack-ecmascript/js/src/runtime.js

```diff
@@ -191,8 +191,7 @@
       // an external module we don't provide a shim for (e.g. querystring, url).
       // For now, we fail semi-silently, but in the future this should be a
       // compilation error.
-      console.error(`Failed to load external module ${id}: ${err}`);
-      return undefined;
+      throw new Error(`Failed to load external module ${id}: ${err}`);
```

Comment:
There might be a reason behind that. See comment. Maybe @alexkirsz knows more?
If we remove that we should also remove the comment

Reply:
I think the comment is about providing a better UX during compilation?

As for the runtime UX, it "semi-silently" fails meaning that as soon as you try to use the return value, you lose the real reason for the error happening. I think loudly failing provides a better experience.